tests/validators/tests.py

```diff
@@ -325,19 +520,10 @@
     (ProhibitNullCharactersValidator(), None, None),
 ]
 
-
-def create_path(filename):
-    return os.path.abspath(os.path.join(os.path.dirname(__file__), filename))
-
-
 # Add valid and invalid URL tests.
 # This only tests the validator without extended schemes.
-with open(create_path('valid_urls.txt'), encoding='utf8') as f:
-    for url in f:
-        TEST_DATA.append((URLValidator(), url.strip(), None))
-with open(create_path('invalid_urls.txt'), encoding='utf8') as f:
-    for url in f:
-        TEST_DATA.append((URLValidator(), url.strip(), ValidationError))
+TEST_DATA.extend(((URLValidator(), url, None) for url in VALID_URLS))
+TEST_DATA.extend(((URLValidator(), url, ValidationError) for url in INVALID_URLS))
```

Comment:
You shouldn't need the extra parentheses inside `extend()`, FYI.

Reply:
I didn't know that. Thanks. I've updated it.